resources/views/nac.blade.php

```diff
@@ -23,8 +23,8 @@
                     <th data-column-id="username" data-formatter="tooltip" data-width="250px">{{ __('Username') }}</th>
                     <th data-column-id="authz_by" data-formatter="tooltip">{{ __('Auth By') }}</th>
                     <th data-column-id="timeout" data-formatter="time_interval">{{ __('Timeout') }}</th>
-                    <th data-column-id="time_elapsed" data-formatter="time_interval" >{{ __('Time Elapsed') }}</th>
```

Comment:
Hi
Your change is not acceptable because some devices provide only time elapsed and not time left. The right way to do it is to change the default value based on OS here.

Reply:
What remains then is to discover both columns or to unhide the columns based on the OS version (each OS version would require separate parameterization)?

Reply:
What remains then is to discover both columns or to unhide the columns based on the OS version (each OS version would require separate parameterization)?

Reply:
You can add conditions in the blade file. So basically, you can consider current display the "default" and add conditions for your OS and change visibility.